app/views/eventDetails/paymentOptions.html

```diff
@@ -61,57 +71,58 @@ <h3 class="panel-title" translate>Checks</h3>
       </div>
       <div class="col-sm-3 stacked-spacing-col-sm">
         <select class="form-control" ng-model="conference.checkMailingState">
-          <option value="AL">Alabama</option>
```

Comment:
Thanks.  I did some research to make sure we could use translate with and select option with gettext but really could not find a lot of documentation with this specific use case.  But also didn't find anything against.